scripts/README.rst

```diff
@@ -0,0 +1,120 @@
+.. _Updating the stm32Cube modules:
+
+Steps to update the stm32Cube modules for zephyrproject
+#######################################################
+
+Overview
+********
+This is a brief description of the different steps to update
+the stm32Cube hal, from the newest STM32CubeXX repositories.
+
+1. modify code
+**************
+
+get STM32CubeXX version
+=======================
+From the ~/zephyrproject/modules/hal/stm32/scripts,
+fetch all the STM32CubeXX directories from the STM, with :
+
+::
+
+ $ python fetch_cubes.py
+
+Then update all the STM32CubeXX directories, with :
+
+::
+
+$ python update_cubes.py
+
+check STM32CubeXX changes
+=========================
+Check that all existing change to modules are correctly reported
+in the new hal_stm32 version
+
+Update each modified stm32cube/stm32XXxx/README in the last section,
+
+Remove the following line "--> please check that the following list is still valid:\n"
+
+       Patch List:
+
+       	See release_note.html from STM32Cube
+
+
+2. create pull requests
+***********************
+Optionnaly:
```

Comment:
Not clear what these optional steps mean

Reply:
Done